### PR TITLE
[ Master Bug 10825 ] - <OTRS_CUSTOMER_Body> in Reply-Template - version 2

### PR DESCRIPTION
**Pull request by s7design**
http://bugs.otrs.org/show_bug.cgi?id=10825

Hi @carlosgarciac and @mgruner 

As it is mentioned on the PR https://github.com/OTRS/otrs/pull/818, I implanted new function in TemplateGenerator for removing unsupported tags. There are unit test for Salutation, Signature and Template.
If you have any suggestion please let me know.

Regards
Zoran

### CODE REVIEW

File: scripts/test/Salutation.t [hunk 1]
```diff
@@ -19,7 +19,7 @@ my $SalutationObject = $Kernel::OM->Get('Kernel::System::Salutation');
 
 # add salutation
 my $SalutationNameRand0 = 'example-salutation' . int rand 1000000;
-my $Salutation          = "Dear <OTRS_CUSTOMER_Realname>,
+my $Salutation          = "Dear <OTRS_CUSTOMER_REALNAME>,
```
Comment (s7design) [inline scripts/test/Salutation.t]: I fixed this tags, in TemplaeGenerator only OTRS_CUSTOMER_REALNAME is supported.

**Comment by s7design** [comment]
As you can see I set only OTRS_AGENT_SUBJECT OTRS_AGENT_BODY OTRS_CUSTOMER_BODY OTRS_CUSTOMER_SUBJECT for each template, but you can suggest me if it could be changed and improved with adding different tags.

Regards
Zoran

**Comment by carlosgarciac** [comment]
Hi @s7design,

could your please resolve conflicts on this branch, then I could merge it.

Best regards,
Carlos García

**Comment by s7design** [comment]
Hi @carlosgarciac ,

This PR is rebased and updated unit tests. Please let me know if you have any further issues.

Regards,
Sanjin

**Comment by s7design** [comment]
Hi @carlosgarciac 

I rebased this PR to make it up to date with the changes which have been happened in meantime.

Regards
Zoran

**Comment by carlosgarciac** [comment]
Hi Zoran,

Thanks so much for your nice effort, you PR was merged in 5855b41.


Best regards,
Carlos García

**Pull request closed without merging**